lightning/src/chain/keysinterface.rs

```diff
@@ -7,9 +7,7 @@
 // You may not use this file except in accordance with one or both of these
 // licenses.
 
-//! keysinterface provides keys into rust-lightning and defines some useful enums which describe
-//! spendable on-chain outputs which the user owns and is responsible for using just as any other
-//! on-chain output which is theirs.
+//! Provides keys to LDK and defines some useful objects describing spendable on-chain outputs.
```

Comment:
"The output descriptor are a LDK-custom data format and are not currently compatible with Bitcoin Core output descriptors. Extensions there could enable wallet compatibility for both on-chain/off-chain descriptors"

FYI, this was the original idea though we never went there! See https://bitcoinops.org/en/topics/output-script-descriptors/ for more. It would be a UX gain for the end users, being able to import/export all your LN channels UTXO from a wallet to another one.

Reply:
Added the first part, the note about possible extensions seemed a bit out of place in the module description.